java/src/jmri/jmrix/can/adapters/gridconnect/lccbuffer/serialdriver/ConnectionConfig.java

```diff
@@ -0,0 +1,36 @@
+package jmri.jmrix.can.adapters.gridconnect.lccbuffer.serialdriver;
+
+/**
+ * Definition of objects to handle configuring a layout connection via a Canusb
+ * SerialDriverAdapter object.
+ *
+ * @author Bob Jacobsen Copyright (C) 2001, 2003
+ * @author Andrew Crosland 2008
+ * @version	$Revision$
+ */
+@edu.umd.cs.findbugs.annotations.SuppressFBWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "name assigned historically")
+public class ConnectionConfig extends jmri.jmrix.can.adapters.ConnectionConfig {
```

Comment:
Is there a reason this class can't be named "LccBufferConnectionConfig"? The "name assigned historically" seems to be a strange justification.